docs/flax.optim.rst

```diff
@@ -2,44 +2,40 @@
 flax.optim package
 ===================
 
-Optimizer core
-------------------------
-
 .. currentmodule:: flax.optim
 
-.. automodule:: flax.optim
-
+Optimizer Base Classes
+------------------------
 
 .. autoclass:: Optimizer
-   :members: apply_gradient, compute_gradients, optimize
+   :members: apply_gradient, replicate, unreplicate
```

Comment:
replicate & unreplicate have been deprecated for a long time so maybe they should not be added to the docs now

Reply:
I forgot about that, and it seems they also don't have a deprecation warning. I added it to both of them and remove them from this list.